### PR TITLE
Add PR Template, CONTRIBUTING.md files

### PR DESCRIPTION
**Pull request by mateuszdyminski**
To make life of our contributors easier

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* Must pass all pre-commit hooks. To install pre-commit hooks, run `make initialise` in the root of the repository
+* Any change to a chart requires a version bump following [semver](https://semver.org/) principles. See [Immutability](#immutability) and [Versioning](#versioning) below
+* Please remember to update the [README.md](./stable/ksoc-plugins/README.md)
+* Please remember to update the [Chart.yaml](./stable/ksoc-plugins/Chart.yaml) with the new version number and update `artifacthub.io/changes` section with the changes made in the chart
+
+Once changes have been merged, the release job will automatically run to package and release changed charts.
+
+### Immutability
+
+Chart releases must be immutable. Any change to a chart warrants a chart version bump even if it is only changed to the documentation.
```
Comment (pancernik) [inline CONTRIBUTING.md:47]: `a change to the documentation`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+1. Remember to sign off your commits as described above
+1. Submit a pull request
+
+***NOTE***: In order to make testing and merging of PRs easier, please submit changes to multiple charts in separate PRs.
+
+### Technical Requirements
+
+* Must pass [DCO check](#sign-off-your-work)
+* Must follow [Charts best practices](https://helm.sh/docs/topics/chart_best_practices/)
+* Must pass all pre-commit hooks. To install pre-commit hooks, run `make initialise` in the root of the repository
```
Comment (pancernik) [inline CONTRIBUTING.md:38]: Probably worth to add it to `How to Contribute`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+### Immutability
+
+Chart releases must be immutable. Any change to a chart warrants a chart version bump even if it is only changed to the documentation.
+
+### Versioning
+
+The chart `version` should follow [semver](https://semver.org/).
+
+Charts should start at `1.0.0`. Any breaking (backwards incompatible) changes to a chart should:
```
Comment (pancernik) [inline CONTRIBUTING.md:53]: Probably worth to suggest that there must be strong reasons for introducing a breaking change and suggest raising an issue explaning the intended change first before the code gets written.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mateuszdyminski) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (mateuszdyminski) [reply CONTRIBUTING.md:38]: Add new section in main README.md

File: CONTRIBUTING.md (hunk 3 above)
Comment (mateuszdyminski) [reply CONTRIBUTING.md:53]: added

**Pull request merged**